tool/intellij/README.md

```diff
@@ -0,0 +1,34 @@
+## Live Templates for MobX
+
+### Add Live Templates
+
+Copy file `MobX.xml` to `AndroidStudio3.4/templates/`
```

Comment:
- You can keep the version part as `<version>` 
- Also can you include the path to Intellij as well ?

Reply:
Looking good!